test/Makefile

```diff
@@ -96,7 +97,7 @@ export REQUIRED_ARGS=
 
 ifeq ($(findstring win,$(OS)),win)
 export ARGS=-inline -release -g -O
-export DMD=../src/dmd.exe
+export DMD=..\generated\$(OS)\$(MODEL)\dmd.exe
```

Comment:
This path should include $(BUILD) as well.  It's a mistake that the current windows make files don't put the binary in the right place and this change shouldn't perpetuate the error.